confignode/src/main/java/org/apache/iotdb/confignode/physical/sys/SetStorageGroupPlan.java

```diff
@@ -18,21 +18,21 @@
  */
 package org.apache.iotdb.confignode.physical.sys;
 
-import org.apache.iotdb.confignode.partition.DataRegionInfo;
-import org.apache.iotdb.confignode.partition.SchemaRegionInfo;
+import org.apache.iotdb.commons.partition.RegionReplicaSet;
 import org.apache.iotdb.confignode.partition.StorageGroupSchema;
 import org.apache.iotdb.confignode.physical.PhysicalPlan;
 import org.apache.iotdb.confignode.physical.PhysicalPlanType;
 
+import java.io.IOException;
 import java.nio.ByteBuffer;
+import java.util.ArrayList;
+import java.util.List;
 
 public class SetStorageGroupPlan extends PhysicalPlan {
```

Comment:
You'd better separate SetStorageGroup with Region Allocation

Reply:
But it is necessary to allocate some Region when creating a StorageGroup, which we've discussed on Friday.

Reply:
fixed